rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -2959,27 +2968,36 @@ error_code sys_fs_get_mount_info(ppu_thread&, vm::ptr<CellFsMountInfo> info, u32
 		sys_fs.todo("sys_fs_get_mount_info special case TODO");
 	}
 
-	const u32 max_len = std::min<u32>(len, 9);
+	const u32 max_len = std::min<u32>(len, lv2_fs_object::get_mount_count());
 	*out_len = max_len;
 
 	struct mount_info
 	{
-		std::string_view path, filesystem, dev_name;
+		std::string path, filesystem, dev_name;
 		be_t<u32> unk1 = 0, unk2 = 0, unk3 = 0, unk4 = 0, unk5 = 0;
 	};
 
-	static constexpr std::array<mount_info, 9> data
-	{
-		mount_info{.path = "/", .filesystem = "CELL_FS_ADMINFS", .dev_name = "CELL_FS_ADMINFS:", .unk5 = 0x10000000},
-		mount_info{.path = "/app_home", .filesystem = "CELL_FS_DUMMY", .dev_name = "CELL_FS_DUMMY:"},
-		mount_info{.path = "/host_root", .filesystem = "CELL_FS_DUMMY", .dev_name = "CELL_FS_DUMMY:"},
-		mount_info{.path = "/dev_flash", .filesystem = "CELL_FS_FAT", .dev_name = "CELL_FS_IOS:BUILTIN_FLSH1:", .unk5 = 0x10000000},
-		mount_info{.path = "/dev_flash2", .filesystem = "CELL_FS_FAT", .dev_name = "CELL_FS_IOS:BUILTIN_FLSH2:"},
-		mount_info{.path = "/dev_flash3", .filesystem = "CELL_FS_FAT", .dev_name = "CELL_FS_IOS:BUILTIN_FLSH3:"},
-		mount_info{.path = "/dev_hdd0", .filesystem = "CELL_FS_UFS", .dev_name = "CELL_FS_UTILITY:HDD0:"},
-		mount_info{.path = "/dev_hdd1", .filesystem = "CELL_FS_FAT", .dev_name = "CELL_FS_UTILITY:HDD1:"},
-		mount_info{.path = "/dev_bdvd", .filesystem = "CELL_FS_ISO9660", .dev_name = "CELL_FS_IOS:PATA0_BDVD_DRIVE"},
-	};
+	std::vector<mount_info> data;
+
+	for (auto mp : mp_array)
+	{
+		if (mp->is_mounted == 0)
+			continue;
+		if (mp == &g_mp_sys_dev_usb)
```

Comment:
It would be appreciated if you could add an empty line between two unrelated if clauses or scopes.
This is not a contest for the most cramped code xD

Reply:
Done.